.changeset/clever-eggs-sit.md

```diff
@@ -0,0 +1,6 @@
+---
+"miniflare": minor
+"wrangler": minor
+---
+
+Add support for streaming tail consumers
```

Comment:
This could do with a bit more content. Perhaps a small example? Or at least a description of what a streaming tail is compared to a tail